jerry-core/ecma/builtin-objects/ecma-builtin-helpers.h

```diff
@@ -93,36 +93,36 @@ typedef enum
 } ecma_date_timezone_t;
 
 /* ecma-builtin-helpers-date.c */
-extern ecma_number_t ecma_date_day (ecma_number_t);
-extern ecma_number_t ecma_date_time_within_day (ecma_number_t);
-extern ecma_number_t ecma_date_days_in_year (ecma_number_t);
-extern ecma_number_t ecma_date_day_from_year (ecma_number_t);
-extern ecma_number_t ecma_date_time_from_year (ecma_number_t);
-extern ecma_number_t ecma_date_year_from_time (ecma_number_t);
-extern ecma_number_t ecma_date_in_leap_year (ecma_number_t);
-extern ecma_number_t ecma_date_day_within_year (ecma_number_t);
-extern ecma_number_t ecma_date_month_from_time (ecma_number_t);
-extern ecma_number_t ecma_date_date_from_time (ecma_number_t);
-extern ecma_number_t ecma_date_week_day (ecma_number_t);
-extern ecma_number_t ecma_date_local_tza ();
-extern ecma_number_t ecma_date_daylight_saving_ta (ecma_number_t);
-extern ecma_number_t ecma_date_local_time (ecma_number_t);
-extern ecma_number_t ecma_date_utc (ecma_number_t);
-extern ecma_number_t ecma_date_hour_from_time (ecma_number_t);
-extern ecma_number_t ecma_date_min_from_time (ecma_number_t);
-extern ecma_number_t ecma_date_sec_from_time (ecma_number_t);
-extern ecma_number_t ecma_date_ms_from_time (ecma_number_t);
-extern ecma_number_t ecma_date_make_time (ecma_number_t, ecma_number_t, ecma_number_t, ecma_number_t);
-extern ecma_number_t ecma_date_make_day (ecma_number_t, ecma_number_t, ecma_number_t);
-extern ecma_number_t ecma_date_make_date (ecma_number_t, ecma_number_t);
-extern ecma_number_t ecma_date_time_clip (ecma_number_t);
-extern ecma_number_t ecma_date_timezone_offset (ecma_number_t);
-
```

Comment:
Yeah, there's no change in that line (I just double checked). Probably some heuristic of diff to form a larger block of changes.